encryption/three-bears/three-bears.yaml

```diff
@@ -1,4 +1,8 @@
 name: Three Bears
+description: >-
+  based on module learning with errors (MLWE) from NewHope and Kyber. The
+  polynomial ring underlying the MLWE module is replaced by integers modulo
+  a generalized Mersenne number.
```

Comment:
@dingens is that ok for you? Or is it too long?

Reply:
how about `based on Module LWE, but the underlying polynomial ring is replaced by integers modulo a generalized Mersenne number`